src/main/java/seedu/address/logic/Messages.java

```diff
@@ -14,11 +14,10 @@ public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "Index provided is exceeds list size.";
```

Comment:
Is the extra "is" intended? Also if the index entered is "-100" it might not exactly be "exceeding list size"